piplicenses.py

```diff
@@ -166,7 +166,7 @@ def get_pkg_included_file(pkg, file_names):
                                                        f))))
          for f in file_names]
         for test_file in patterns:
-            if os.path.exists(test_file):
+            if os.path.exists(test_file) and os.path.isdir(test_file) is not True:
```

Comment:
We continue to keep our code in line with PEP8.

This line should satisfy Style Guide PEP8,

e.g.)
```py
if os.path.exists(test_file) and \            
        os.path.isdir(test_file) is not True: 
```